src/Compilers/Core/Portable/Collections/ArrayBuilderExtensions.cs

```diff
@@ -173,5 +173,14 @@ public static void AddIfNotNull<T>(this ArrayBuilder<T> builder, T value)
                 builder.Add(value);
             }
         }
+
+        public static void FreeAll<T>(this ArrayBuilder<T> builder, Func<T, ArrayBuilder<T>> getNested)
+        {
+            foreach (var item in builder)
+            {
+                getNested(item)?.Free();
```

Comment:
>Free [](start = 33, length = 4)

Should this be `FreeAll` too?

Reply:
Good catch. Thanks.

---
In reply to: [252777540](https://github.com/dotnet/roslyn/pull/32994#discussion_r252777540) [](ancestors = 252777540)